containers/sfdx-project/.npmignore

```diff
@@ -0,0 +1,3 @@
+README.md
```

Comment:
@dehru - Any reason we need a .npmignore? Is this repository being packaged as an NPM module?